airbnb_project/database/airbnb_db_creator.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE IF NOT EXISTS listing (
+    airbnb_listing_id VARCHAR(20) PRIMARY KEY,
+    baths INT,
+    beds INT,
+    latitude DECIMAL(8,6), 
+    location VARCHAR(20),
+    longitude DECIMAL(9,6),
+    name VARCHAR(100),
+    person_capacity INT,
+    registration_number VARCHAR(20),
+    room_type VARCHAR(20),
+    title VARCHAR(100)
+);
+
+CREATE TABLE IF NOT EXISTS policy (
+    policy_id SERIAL PRIMARY KEY,
+    policy_name VARCHAR(255),
+    policy_description VARCHAR(255)
+);
+
+CREATE TABLE IF NOT EXISTS listing_policy_result (
+    result_id SERIAL PRIMARY KEY,
+    airbnb_listing_id VARCHAR(50),
+    policy_id INT,
+    policy_result BOOLEAN,
+    result_details VARCHAR(255),
+    result_datetime TIMESTAMP,
+    FOREIGN KEY (airbnb_listing_id) REFERENCES listing(airbnb_listing_id),
+    FOREIGN KEY (policy_id) REFERENCES policy(policy_id)
+);
+
+INSERT INTO listing (airbnb_listing_id, baths, beds, latitude, location, longitude, name, person_capacity, registration_number, room_type, title) VALUES
```

Comment:
mostly correct, new column is needed and format of room type needs to be changed. 

Some sample data: 

|airbnb_listing_id  |bath_is_shared|baths|baths_text|beds|latitude         |location |longitude          |name                                            |person_capacity|registration_number|room_type   |title                 |
|-------------------|--------------|-----|----------|----|-----------------|---------|-------------------|------------------------------------------------|---------------|-------------------|------------|----------------------|
|1161822129037475555|False         |2    |2 baths   |3   |49.25234553007443|Vancouver|-123.22965155214942|Wesbrook Apartment close to UBC                 |6              |Exempt             |entire_home |Apartment in Vancouver|
|976273126562399664 |False         |1.5  |1.5 baths |1   |49.24029         |Vancouver|-123.19577         |Private Room w/Queen Bed in VancouverWest Dunbar|2              |24-160007          |private_room|Room in Vancouver     |